bundles/org.openhab.core.model.sitemap/src/org/openhab/core/model/sitemap/Sitemap.xtext

```diff
@@ -146,7 +147,7 @@ Mapping:
     cmd=Command '=' label=(ID | STRING);
 
 VisibilityRule:
-    (item=ID) (condition=("==" | ">" | "<" | ">=" | "<=" | "!=")) (sign=('-' | '+'))? (state=XState);
+    (item=ID) (condition=('==' | '>' | '<' | '>=' | '<=' | '!=')) (sign=('-' | '+'))? (state=XState);
```

Comment:
Why did you change this?

Reply:
Purely for consistency, so cleanup. Xtext will allow single or double quotes. All other matches in the file use single quotes. There is no specific reason for the functionality of this PR.

Reply:
Agree with @mherwege

Reply:
@J-N-K @lolodomo That is correct. I intended to do that in a PR related to sitemap config in mainUI, because my understanding is that is where it is used. See https://github.com/openhab/openhab-core/pull/3431 for the base one (without inputHint). My intention was to create a separate PR for this, but I can include it in the existing open one if preferred.